prow/plugins/lgtm/lgtm.go

```diff
@@ -204,3 +207,26 @@ func handle(gc githubClient, log *logrus.Entry, e *event) error {
 	}
 	return nil
 }
+
+func handlePullRequest(gc githubClient, pe github.PullRequestEvent) error {
+	if pe.PullRequest.Merged {
+		return nil
+	}
+
+	switch pe.Action {
+	case "synchronize":
+		// Don't bother checking if it has the label...it's a race, and we'll have
+		// to handle failure due to not being labeled anyway.
+		if err := gc.RemoveLabel(
+			pe.PullRequest.Base.Repo.Owner.Login,
+			pe.PullRequest.Base.Repo.Name,
+			pe.PullRequest.Number,
+			lgtmLabel,
+		); err != nil {
+			// TODO(nlandolfi): inspect this error?
```

Comment:
returns a 404 Label does not exist, but not sure how well exposed this is when the RemoveLabel returns. I could inspect the error text specifically, but the raw error interface doesn't give me much info...looks like the error code will be something like:

`status code 404 not one of [200 204], body: Label not found`

alternative is to add 404 to list of "ok" return codes in RemoveLabel...